GitExtensions/GitExtensions.csproj

```diff
@@ -18,6 +18,9 @@
 
     <!-- To be removed when NRT annotations are complete -->
     <Nullable>annotations</Nullable>
+
+    <!-- Set version for runtimeconfig.json, used by DotnetRuntimeBootstrapper -->
+    <RuntimeFrameworkVersion>6.0.8</RuntimeFrameworkVersion>
```

Comment:
I suggest moving this to here, so the sdk/runtime requirements are all in the same place
https://github.com/gitextensions/gitextensions/blob/37797ea4dd74c57d223c44a0ccddc54e3b091319/scripts/RepoLayout.props#L9

Reply:
Seem to work